packages/orchestration/src/orchestration-client.ts

```diff
@@ -36,7 +36,11 @@ export class OrchestrationClient {
     prompt?: Prompt,
     requestConfig?: CustomRequestConfig
   ): Promise<OrchestrationResponse> {
-    const body = constructCompletionPostRequest(this.config, prompt);
+    const body: CompletionPostRequest | Record<string, any> =
+      typeof this.config === 'string'
+        ? constructCompletionFromJson(this.config, prompt)
```

Comment:
[pp] Can we rename this into `constructCompletionPostRequestFromJson`? Construct completion sounds pretty weird to me. 
[pp] Or maybe put the type check into the original `constructCompletionPostRequest` function.

Reply:
Like the name `constructCompletionPostRequestFromJson`. Is clearer. I would keep the type of check here though.